Sources/KSCrashRecording/Monitors/KSCrashMonitor_CPPException.cpp

```diff
@@ -145,34 +159,37 @@ static void CPPExceptionTerminate(void)
         memset(crashContext, 0, sizeof(*crashContext));
 
         char descriptionBuff[DESCRIPTION_BUFFER_LENGTH];
-        description = descriptionBuff;
-        descriptionBuff[0] = 0;
-
-        KSLOG_DEBUG("Discovering what kind of exception was thrown.");
-        g_captureNextStackTrace = false;
-        try {
-            throw;
-        } catch (std::exception &exc) {
-            strncpy(descriptionBuff, exc.what(), sizeof(descriptionBuff));
-        }
+        if (!emptyThrow) {
+            description = descriptionBuff;
+            descriptionBuff[0] = 0;
```

Comment:
If we pre-populate the `descriptionBuff` with a `{0}` this line isn't required.

Reply:
done